Cargo.toml

```diff
@@ -37,3 +37,6 @@ pr-run-mode = "upload"
 # This should not be necessary once the following issue is addressed:
 # https://github.com/axodotdev/cargo-dist/issues/423
 allow-dirty = ["ci"]
+
+[patch.crates-io]
+borsh = { git = 'https://github.com/near/borsh-rs.git', branch = "slice_schema_container" }
```

Comment:
TODO: remove after [borsh#245](https://github.com/near/borsh-rs/pull/245) is merged